pkg/acceptance/chaos_test.go

```diff
@@ -254,7 +254,9 @@ func chaosMonkey(
 			time.Sleep(time.Second)
 		}
 		c.Assert(ctx, state.t)
-		cluster.Consistent(ctx, state.t, c)
+		if err := cluster.Consistent(ctx, c); err != nil {
+			state.t.Error(err)
```

Comment:
Note that this differs from the old behaviour which called `Fatal`, but this change is necessary due to the unsafety of calling `Fatal` in the non-main goroutine now being detected by staticcheck.

I've filed an issue upstream: https://github.com/dominikh/go-staticcheck/issues/60 to track the naivete if staticcheck in this regard.